src/modules/setup.js

```diff
@@ -6,6 +6,18 @@ async function firstTimeSetup(redisURL) {
     let conn = new Redite({url: redisURL});
 
     if (await conn.has('NOT_FIRST_SETUP')) return;
+    if (process.env.KNIFE_IN_DOCKER === 'true') {
+        console.log('Knife is in Docker. Setting values automatically.');
+        await conn.NOT_FIRST_SETUP.set(true);
+        await conn.settings.prefixes.set([process.env.KNIFE_PREFIX].concat(['<@{{id}}> ', '<@!{{id}}>']));
+        await conn.settings.token.set(process.env.KNIFE_DISCORD_TOKEN);
+        await conn.settings.owner.set(process.env.OWNER);
+        await conn.settings.blacklist.set([]);
+        await conn.settings.unloaded.set([]);
+        if (process.env.KNIFE_DBOTS_TOKEN) await conn.settingsdbotsToken.set(process.env.KNIFE_DBOTS_TOKEN);
+        
+        return;
+    } else return;
```

Comment:
Remove this `else`, otherwise nothing will run if the env var isn't set.

Reply:
hol up :b: I already removed this